README.md

```diff
@@ -15,13 +15,9 @@ Speakerfight is an arena where speakers can fight ~against~ each other and peopl
 $ git clone git@github.com:luanfonceca/speakerfight.git
 $ cd speakerfight
 $ pip install -r requirements.txt
```

Comment:
I think it should stay the way it is now! ```make deps``` will make people believe that they didn't install any other dependencies!